druid/src/widget/svg.rs

```diff
@@ -373,7 +373,13 @@ impl SvgRenderer {
                 let color = color_from_svg(*c, opacity);
                 Rc::new(ctx.solid_brush(color))
             }
-            usvg::Paint::Link(id) => self.defs.find(id).unwrap(),
+            usvg::Paint::Link(id) => match self.defs.find(id) {
+                Some(v) => v,
+                None => {
+                    trace!("svg link id requested but not found: {:?}", id);
```

Comment:
warn or error?

Reply:
indifferent, I ran into this case on files in the wild. personally I just think its a fact of life and not really a error or issue.

Reply:
thinking on it more, I didn't look too closely at the svg... and looking at druids code it code have been due to another unimplemented node in the svg tree. ;) i think in that context probably should move it to a warn.

Reply:
trace will go unnoticed during development. but warn can be spammy I believe :/

Reply:
I did confirm that none can occur due to unimplemented nodes in druid. as for spammy, svg stuff already spams like crazy due to missing implementation details.

anyways, logging can be adjusted at any time.  atm I think this is fine until SVG is more fully implemented. its more important that druid doesn't crash as a result.